views/layouts/main.handlebars

```diff
@@ -43,12 +43,12 @@
   </div>
   <div class="footer">
     <div class="legal-links">
-      <a href="https://www.mozilla.org"><img class="mozilla-logo" src="/resources/mozilla-logo.svg"/></a>
-      <a href="https://www.mozilla.org/about/legal" data-l10n-id="footerLinkLegal">Legal</a>
-      <a href="https://testpilot.firefox.com/about" data-l10n-id="footerLinkAbout">About Test Pilot</a>
-      <a href="/legal" data-l10n-id="footerLinkPrivacy">Privacy</a>
-      <a href="/legal" data-l10n-id="footerLinkTerms">Terms</a>
-      <a href="https://www.mozilla.org/privacy/websites/#cookies" data-l10n-id="footerLinkCookies">Cookies</a>
+      <a href="https://www.mozilla.org" target="_blank"><img class="mozilla-logo" src="/resources/mozilla-logo.svg"/></a>
+      <a href="https://www.mozilla.org/about/legal" data-l10n-id="footerLinkLegal" target="_blank">Legal</a>
+      <a href="https://testpilot.firefox.com/about" data-l10n-id="footerLinkAbout" target="_blank">About Test Pilot</a>
+      <a href="/legal" data-l10n-id="footerLinkPrivacy" target="_blank">Privacy</a>
+      <a href="/legal" data-l10n-id="footerLinkTerms" target="_blank">Terms</a>
+      <a href="https://www.mozilla.org/privacy/websites/#cookies" data-l10n-id="footerLinkCookies" target="_blank">Cookies</a>
```

Comment:
The only one without it now is the Send logo. I'd kinda like to keep it like that